src/lib/machines/reagentTransfer/reagentTransferMachine.ts

```diff
@@ -150,6 +162,9 @@ export const reagentTransferMachine = createMachine<ReagentTransferContext, Reag
           UPDATE_TRANSFER_CONTENT: {
             actions: ['assignTransfers']
```

Comment:
Unrelated to the PR but some of these actions are in lists and others are not, probably good practice to standardise it so all are in lists or none are in lists.

Reply:
Done